src/nft/ZoraCreator1155Impl.sol

```diff
@@ -444,15 +445,38 @@ contract ZoraCreator1155Impl is
         // Require admin from the minter to mint
         _requireAdminOrRole(address(minter), tokenId, PERMISSION_BIT_MINTER);
 
+        _handleFirstMinter(tokenId, minterArguments);
+
         // Get value sent and handle mint rewards
-        uint256 ethValueSent = _handleRewardsAndGetValueSent(msg.value, quantity, getCreatorRewardRecipient(), createReferrals[tokenId], mintReferral);
+        uint256 ethValueSent = _handleRewardsAndGetValueSent(
+            msg.value,
+            quantity,
+            getCreatorRewardRecipient(),
+            createReferrals[tokenId],
+            mintReferral,
+            firstMinters[tokenId]
```

Comment:
seems like we are double looking up `firstMinters[tokenId]` here and in `_handleFirstMinter` - can this be reduced to one storage lookup?

Reply:
https://github.com/ourzora/zora-1155-contracts/pull/162/files#diff-2a72e8508d3c8b8b46fde1dda4e0f8720176a5839bfb9c1490f38173325d2687R469-R483